src/jira/client/axios.ts

```diff
@@ -133,6 +134,8 @@ const instrumentRequest = (response) => {
 		status: response.status
 	};
 
+	statsd.histogram(metricHttpRequest.jira, requestDurationMs, tags);
+	tags["gsd_histogram"] = RESPONSE_TIME_HISTOGRAM_BUCKETS;
 	statsd.histogram(metricHttpRequest.jira, requestDurationMs, tags);
```

Comment:
Posting the same metric with `gsd_histogram` to produce a bucketed histogram metric. (Like we do for GH client and server side request metrics)